pkg/operator/starter.go

```diff
@@ -301,6 +302,20 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		return !isSNO, precheckSucceeded, err
 	}
 
+	// checks if quorum is valid
+	quorumSafe := func() (bool, error) {
```

Comment:
great start! could you also add another commit with the removal of all existing checks? That way we can maybe have an easier time to test whether it works correctly on its own.

Reply:
@tjungblu I have now added a commit removing the existing checks(with commenting the corresponding test routine changes for now). There are two required cases that are failing now.